### PR TITLE
add cluster-autoscaler missing flags in documentation

### PR DESCRIPTION
**Pull request by yogeek**
Signed-off-by: yogeek <gdupin@gmail.com>

#### Which component this PR applies to?

cluster-autoscaler

#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

The flags added in #1407 were missing from the documentation. This PR adds them.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: cluster-autoscaler/FAQ.md [hunk 1]
```diff
@@ -532,7 +532,7 @@ any nodes left unregistered after this time.
 Every 10 seconds (configurable by `--scan-interval` flag), if no scale-up is
 needed, Cluster Autoscaler checks which nodes are unneeded. A node is considered for removal when **all** below conditions hold:
 
-* The sum of cpu and memory requests of all pods running on this node is smaller
+* The sum of cpu and memory requests of all pods running on this node (DaemonSet pods and Mirror pods are included by default but this is configurable with `--ignore-daemonsets-utilization` and `ignore-mirror-pods-utilization` flags) is smaller
```
Comment (elmiko) [inline cluster-autoscaler/FAQ.md:535]: just a nit
```suggestion
* The sum of cpu and memory requests of all pods running on this node (DaemonSet pods and Mirror pods are included by default but this is configurable with `--ignore-daemonsets-utilization` and `--ignore-mirror-pods-utilization` flags) is smaller
```

**Review by elmiko — COMMENTED** [review]
thanks for the contribution, this is a nice update. i have one minor suggestion to make it more consistent, otherwise it looks good to me.

**Comment by yogeek** [comment]
Thank you for the review @elmiko 
Just updated the PR with the little fix

**Review by elmiko — COMMENTED** [review]
thanks!
/lgtm

**Review by mwielgus — APPROVED** [review]
/lgtm
/approve

**Pull request merged**